server/src/main/java/org/elasticsearch/index/engine/Engine.java

```diff
@@ -174,6 +176,10 @@ public MergeStats getMergeStats() {
     /** returns the history uuid for the engine */
     public abstract String getHistoryUUID();
 
+    /** returns the force merge uuid for the engine */
+    @Nullable
+    public abstract String getForceMergeUUID();
```

Comment:
Can we move this method to InternalEngine?

Reply:
Done :)